deploy/helm/sumologic/values.yaml

```diff
@@ -729,24 +770,83 @@ prometheus-operator:
+        # node:node_memory_utilisation:ratio
+        # node:node_memory_utilisation_2:
+        # node:node_net_saturation:sum_irate
+        # node:node_net_utilisation:sum_irate
+        # node:node_num_cpu:sum
+        # node_namespace_pod:kube_pod_info:
         - url: http://$(CHART).$(NAMESPACE).svc.cluster.local:9888/prometheus.metrics.operator.rule
           writeRelabelConfigs:
             - action: keep
               regex: 'cluster_quantile:apiserver_request_latencies:histogram_quantile|instance:node_filesystem_usage:sum|instance:node_network_receive_bytes:rate:sum|cluster_quantile:scheduler_e2e_scheduling_latency:histogram_quantile|cluster_quantile:scheduler_scheduling_algorithm_latency:histogram_quantile|cluster_quantile:scheduler_binding_latency:histogram_quantile|node_namespace_pod:kube_pod_info:|:kube_pod_info_node_count:|node:node_num_cpu:sum|:node_cpu_utilisation:avg1m|node:node_cpu_utilisation:avg1m|node:cluster_cpu_utilisation:ratio|:node_cpu_saturation_load1:|node:node_cpu_saturation_load1:|:node_memory_utilisation:|node:node_memory_bytes_total:sum|node:node_memory_utilisation:ratio|node:cluster_memory_utilisation:ratio|:node_memory_swap_io_bytes:sum_rate|node:node_memory_utilisation:|node:node_memory_utilisation_2:|node:node_memory_swap_io_bytes:sum_rate|:node_disk_utilisation:avg_irate|node:node_disk_utilisation:avg_irate|:node_disk_saturation:avg_irate|node:node_disk_saturation:avg_irate|node:node_filesystem_usage:|node:node_filesystem_avail:|:node_net_utilisation:sum_irate|node:node_net_utilisation:sum_irate|:node_net_saturation:sum_irate|node:node_net_saturation:sum_irate|node:node_inodes_total:|node:node_inodes_free:'
               sourceLabels: [__name__]
         # health
+        # prometheus_remote_storage_dropped_samples_total
+        # prometheus_remote_storage_enqueue_retries_total
+        # prometheus_remote_storage_failed_samples_total
+        # prometheus_remote_storage_highest_timestamp_in_seconds
+        # prometheus_remote_storage_pending_samples
+        # prometheus_remote_storage_queue_highest_sent_timestamp_seconds
+        # prometheus_remote_storage_retried_samples_total
+        # prometheus_remote_storage_samples_in_total
+        # prometheus_remote_storage_sent_batch_duration_seconds_bucket
+        # prometheus_remote_storage_sent_batch_duration_seconds_count
+        # prometheus_remote_storage_sent_batch_duration_seconds_sum
+        # prometheus_remote_storage_shard_capacity
+        # prometheus_remote_storage_shards
+        # prometheus_remote_storage_shards_desired
+        # prometheus_remote_storage_shards_max
+        # prometheus_remote_storage_shards_min
+        # prometheus_remote_storage_string_interner_zero_reference_releases_total
+        # prometheus_remote_storage_succeeded_samples_total
+        # up
         - url: http://$(CHART).$(NAMESPACE).svc.cluster.local:9888/prometheus.metrics
           writeRelabelConfigs:
             - action: keep
```

Comment:
missing fluentd/bit/otel?

Reply:
+1

Reply:
Yeah, I was basing on metrics test. Need to run setup with logs and tracing to do full completion